rfc/2019-12-9-pingcap-university-campus.md

```diff
@@ -0,0 +1,65 @@
+- Last updated: 2019-12-6
+- Discussion at:
+
+## Abstract
+
+There are plenty of people interested in TiDB but lack enough background knowledge, especially college students and engineers without backend development experience. The proposal aims at helping those people enter the field of TiDB by creating a series of courses, which is also called Talent Plan Courses. Participants should be able to pull requests to TiDB related projects after consumed the course materials.
+
+## Background
+
+Since the campus is always an important member of the TiDB Community, and we hope more young blood from universities or other fields can embrace the TiDB ecosystem, so we would develop a distributed database system course to help people without enough background knowledge to enter the field. The courses will be consist of reading supplies and programming assignments. As there are already many accessible materials online, we can develop the content of the course by learning from them. We think the voice from the different populations is important for the success of the goal, especially college students, so we hope they can join us to make this happen.
+
+## Proposal
+
+The course is consists of two parts, reading materials and programming assignments. The reading materials can be collected from textbooks, slides, and notes from other related courses.
+
+### Reading Materials
+
+The list of reference for reading materials, both for database system and distributed system:
+
+- [Designing Data-Intensive Application](https://dataintensive.net/)
+- [Reading in Database system, 5th Edition](http://www.redbook.io/)
+- [Architecture of a Database system](http://db.cs.berkeley.edu/papers/fntdb07-architecture.pdf)
+- [Database System Concept](https://www.db-book.com/db6/index.html)
+- [Database System Implementation](http://infolab.stanford.edu/~ullman/dbsi.html)
+- [CMU cs15445 Database System](https://15445.courses.cs.cmu.edu/fall2019/schedule.html)
+- [CMU cs15721 Advanced Database System](https://15721.courses.cs.cmu.edu/spring2019/schedule.html)
+- [UCB cs186 course materials](https://cs186berkeley.net/)
+- [MIT 6.830 course materials](http://db.csail.mit.edu/6.830/)
+- [Stanford cs245 course materials](http://web.stanford.edu/class/cs245/)
+- [MIT 6.824 Distributed Systems](https://pdos.csail.mit.edu/6.824/)
+- [Princeton COS418 Distributed Systems](https://www.cs.princeton.edu/courses/archive/fall19/cos418/)
+- [Princeton COS316 Principle of Computer System Design](https://www.cs.princeton.edu/courses/archive/fall19/cos316/)
+- [UCB Operating Systems and System Programming](https://cs162.eecs.berkeley.edu/)
+- [Distributed System 3rd Edition](https://www.distributed-systems.net/)
+
+Many important topics have already been covered in the above. Course development participants are required to reorganize the content into slides or notes and translate them into Chinese, We hope the final content should be concrete and easy to understand, related to TiDB is preferred.
```

Comment:
```suggestion
Many important topics have already been covered in the above topics, based on which course developers design the course content. The final content should be concrete and easy to understand, and, preferably, related to TiDB.
```

Reply:
Why require translating into Chinese? I don't think it's necessary, as you are writing this in English.